NEWS.md

```diff
@@ -1,5 +1,17 @@
 # bslib (development version)
 
```

Comment:
## Breaking changes

* `value_box()` no longer defaults to `theme_color = "primary"`. To get the old styling back, do `theme = "primary"`.